### PR TITLE
updated polytopes and other files

### PR DESCRIPTION
**Pull request by KayzeTheGeek**

### CODE REVIEW

File: Todo/Polytope.ts [hunk 1]
```diff
@@ -208,13 +210,13 @@ export class SkewPolyhedron {
...
-    ;[-0.08, 0.04, 0.93, -0.36]
-    ;[-0.37, -0.72, 0.21, 0.54]
+      ;[0.62, -0.66, -0.07, -0.41]
+      ;[0.68, 0.22, 0.3, 0.64]
+      ;[-0.08, 0.04, 0.93, -0.36]
+      ;[-0.37, -0.72, 0.21, 0.54]
 
     m = Matrix4D.GramSchmidt(m)
-    for (let poly: Polygon in result) {
+    for (let poly in result) {
```
Comment (lancejpollard) [inline Todo/Polytope.ts]: Should this be `of`?

File: Todo/Polytope.ts [hunk 2]
```diff
@@ -240,34 +239,34 @@ export class SkewPolyhedron {
...
       //  This is important for the Distinct check below.
-      cycle.RemoveAt(cycle.Count - 1)
+      cycle.RemoveAt(cycle.length - 1)
     }
 
     cycles = cycles.Distinct(new CycleEqualityComparer())
     //  Now turn into polygons.
     let result: Array<Polygon> = new Array<Polygon>()
-    for (let cycle: Array<number> in cycles) {
+    for (let cycle in cycles) {
```
Comment (lancejpollard) [inline Todo/Polytope.ts]: Same here.

File: mesh.cs [hunk 3]
```diff
@@ -0,0 +1,377 @@
+namespace R3.Geometry
```
Comment (lancejpollard) [inline mesh.cs]: We should probably delete this file.

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,3571 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (lancejpollard) [inline yarn.lock:2]: Can you describe what changes you made to the yarn file?

File: Todo/Polytope.ts [hunk 5]
```diff
@@ -208,13 +210,13 @@ export class SkewPolyhedron {
     //  Nice starting orientation.
     let m: Matrix4D = new Matrix4D()
     m.Data = []
-    ;[0.62, -0.66, -0.07, -0.41]
-    ;[0.68, 0.22, 0.3, 0.64]
-    ;[-0.08, 0.04, 0.93, -0.36]
-    ;[-0.37, -0.72, 0.21, 0.54]
+      ;[0.62, -0.66, -0.07, -0.41]
```
Comment (lancejpollard) [inline Todo/Polytope.ts]: @KayzeTheGeek what is this, shoudn't this be converted into something useful?

File: yarn.lock (hunk 4 above)
Comment (lancejpollard) [reply yarn.lock:2]: cc @KayzeTheGeek

File: Todo/Polytope.ts (hunk 1 above)
Comment (KayzeTheGeek) [reply Todo/Polytope.ts]: Yes it should be, already fixed that

File: yarn.lock (hunk 4 above)
Comment (KayzeTheGeek) [reply yarn.lock:2]: No changes was made

**Review by lancejpollard — APPROVED** [review]
Excellent work, thank you so much!!!

**Pull request merged**